di_website/visualisation/mixins.py

```diff
@@ -31,6 +31,7 @@ class Meta:
     instructions_heading = models.TextField(
         blank=True, default='Interactive visualisation instructions',
         verbose_name='Accordion heading')
+    instruction_position = models.CharField(max_length=256, choices=POSITION_CHOICES, default='top', verbose_name='Position')
```

Comment:
Should default to bottom to not affect existing charts.

Reply:
Updated